[2025-10-17T08:43:04.487+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-17T08:43:04.506+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:42:00+00:00 [queued]>
[2025-10-17T08:43:04.513+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:42:00+00:00 [queued]>
[2025-10-17T08:43:04.514+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-17T08:43:04.526+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-10-17 08:42:00+00:00
[2025-10-17T08:43:04.535+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=125) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-17T08:43:04.537+0000] {standard_task_runner.py:63} INFO - Started process 129 to run task
[2025-10-17T08:43:04.537+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-10-17T08:42:00+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmp2xfjov5u']
[2025-10-17T08:43:04.541+0000] {standard_task_runner.py:91} INFO - Job 216: Subtask task_pwd
[2025-10-17T08:43:04.558+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-17T08:43:04.606+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:42:00+00:00 [running]> on host 5b9dfa83b040
[2025-10-17T08:43:04.682+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T08:42:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-17T08:42:00+00:00'
[2025-10-17T08:43:04.683+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-17T08:43:04.684+0000] {logging_mixin.py:188} INFO - 1
[2025-10-17T08:43:04.684+0000] {logging_mixin.py:188} INFO - 2
[2025-10-17T08:43:04.684+0000] {logging_mixin.py:188} INFO - 3
[2025-10-17T08:43:04.685+0000] {logging_mixin.py:188} INFO - 4
[2025-10-17T08:43:04.685+0000] {logging_mixin.py:188} INFO - 5
[2025-10-17T08:43:04.685+0000] {logging_mixin.py:188} INFO - 6
[2025-10-17T08:43:04.685+0000] {logging_mixin.py:188} INFO - 7
[2025-10-17T08:43:04.685+0000] {logging_mixin.py:188} INFO - 8
[2025-10-17T08:43:04.686+0000] {logging_mixin.py:188} INFO - 9
[2025-10-17T08:43:04.686+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-17T08:43:04.686+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-17T08:43:04.694+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-10-17T08:42:00+00:00, execution_date=20251017T084200, start_date=20251017T084304, end_date=20251017T084304
[2025-10-17T08:43:04.712+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-17T08:43:04.725+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-17T08:43:04.727+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
